### PR TITLE
8.0 Add "Aged Open Invoices" report in account_financial_report_webkit

### PR DESCRIPTION
**Pull request by alexis-via**
and some string updates on aged balance.

### CODE REVIEW

File: account_financial_report_webkit/report/aged_open_invoices.py [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+
+        res = super(AccountAgedOpenInvoicesWebkit, self).set_context(
+            objects,
+            data,
+            ids,
+            report_type=report_type
+        )
+
+        aged_open_inv = {}
+        # {account1_id: {(0, 30): 0.0,
```
Comment (pedrobaeza) [inline account_financial_report_webkit/report/aged_open_invoices.py]: Remove this commented block

**Comment by pedrobaeza** [comment]
A little PEP8 remark:

```
./account_financial_report_webkit/report/aged_open_invoices.py:197:9: E265 block comment should start with '# '
```

**Comment by pedrobaeza** [comment]
Tried on runbot and seems OK, except 2 little things, :+1:

**Comment by alexis-via** [comment]
@pedrobaeza It seems I missed your comments because they were made during my summer holidays.

I took into accout your remarks.

I deployed this PR with the "aged open invoices" report in the Odoo system of all my customers who use v8 in production with accounting, and they are happy with it.

**Comment by pedrobaeza** [comment]
flake8 error unrelated, so :+1:

**Comment by fclementic2c** [comment]
Is this not similar to https://github.com/OCA/account-financial-reporting/pull/118 (in v7)?

**Comment by alexis-via** [comment]
Yes, it seems similar. My PR is more ancient and is for v8, but the feature seems to be the same.

**Comment by pedrobaeza** [comment]
I merged this old one that is approved. Maybe the scope of only invoices instead of account move lines is more limited, but it can be extended later.

**Pull request merged**